src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -91,14 +90,19 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
             <div className="profile-extras">
               {venue?.profile_questions?.map((question) => (
                 <React.Fragment key="question.text">
-                  <p className="light question">{question.text}</p>
-                  <h6>
-                    {/*
-                    // @debt typing - need to support known User interface with unknown question keys
-                    // @ts-ignore */}
-                    {selectedUserProfile[question.name] || //@debt typing - look at the changelog, was this a bug?
-                      DEFAULT_EDIT_PROFILE_TEXT}
```

Comment:
Here, rather than showing a default text such as 'user hasn't answered yet' beneath the custom profile field, the field name will not be shown if not answered. Thus, no need for a default response.

Reply:
As per 
![image](https://user-images.githubusercontent.com/63194656/120862616-e94d0200-c591-11eb-9729-6c125c95ed7c.png)

I think it's a correct behaviour?